.github/workflows/build.yml

```
@@ -74,11 +75,20 @@ jobs:
         if: matrix.php-version == '8.0'
         run: composer update --ignore-platform-reqs --prefer-dist --no-interaction --no-progress --optimize-autoloader
 
-      - name: Run tests with phpunit
+      - name: Run tests with phpunit and coverage on Linux php 7.4
+        if: matrix.os == 'ubuntu-latest' && matrix.php-version == '7.4'
         run: vendor/bin/phpunit --coverage-clover=coverage.clover
 
-      - name: Code coverage
-        if: matrix.os == 'ubuntu-latest'
+      - name: Run tests with phpunit without coverage on Linux php 8.0
+        if: matrix.os == 'ubuntu-latest' && matrix.php-version == '8.0'
+        run: vendor/bin/phpunit
+
+      - name: Run tests with phpunit without coverage on Windows
+        if: matrix.os == 'windows-latest'
+        run: vendor/bin/phpunit
```

I think it should be merged
```suggestion
      - name: Run tests with phpunit without coverage
        run: vendor/bin/phpunit
```